04-HTML5-APIs/Ex02/script.js

```diff
@@ -0,0 +1,34 @@
+function dragOver(e) {
+	e.preventDefault();
+	e.stopPropagation();
+	e.dataTransfer.dropEffect = 'copy';
+}
+function dropFiles(e) {
+	e.preventDefault();
+	e.stopPropagation();
+	let files = e.dataTransfer.files;
+	var file = files[0];
+	if (file.type == 'text/plain') {
+		function readFile(f) {
```

Comment:
A function inside a function is not good. Move the function declaration outside and just use it here. Same for the function readImage